docs/topics/azure-stack.md

```diff
@@ -15,25 +15,23 @@
 * [Known Issues and Limitations](#known-issues-and-limitations)
 * [Frequently Asked Questions](#frequently-asked-questions)
 
-## Introduction
+## Introduction to the AKS Engine for Azure Stack
```

Comment:
The documentation in general doesn't use "the" before "AKS Engine."

Reply:
@mboersma oh, like the "Titanic" rather than "The Titanic"... thanks!